src/api/app/models/group.rb

```diff
@@ -5,6 +5,7 @@
 # Groups have an arbitrary number of roles and users assigned to them.
 #
 class Group < ApplicationRecord
+  has_one :staging_workflow, foreign_key: :managers_id, dependent: :nullify
```

Comment:
As @hennevogel wrote in https://github.com/openSUSE/open-build-service/pull/6257/commits/d867d2953e4f25038f9ff9d08670074c2d609c8a, this is not needed.

Reply:
But if the admin removes the group? what happen with the staging workflow? it will have an id for a group that doesn't exist

Reply:
True, `:nullify` makes sense. Did not think about this.